### ISSUE TITLE
Allow passing binary and lengthy data as function call arguments

### ISSUE THREAD

**Issue by frol** [near/near-cli-rs#114]
Currently, you can only pass Unicode text as function call arguments (we use `String`), but sometimes there is a need to pass a lengthy JSON or even WASM file (e.g. upgrading the code in factory contracts). I suggest we enable these cases by asking people to choose between:

```
> How do you want to pass the function call arguments?
* json-args    - valid JSON arguments (e.g. {"token_id": "42"})
* text-args    - arbitrary text arguments
* base64-args  - Base64-encoded string (e.g. e30=)
* file-args    - Read from file (e.g. reusable JSON or binary data)
```

The final command could look like this:

```
./near-cli contract \
    call-function \
    as-transaction \
    turbo.volodymyr.testnet \
    'rate'
    json-args '{"other_user":"volodymyr.testnet", "vote":5}' \
        --prepaid-gas '3 Tgas' \
        --attached-deposit '1 NEAR' \
    fro_volod.testnet \
    network testnet \
    sign-with-keychain \
    send
```

Yet, `--prepaid-gas` and `--attached-deposit` look odd this way, so I am considering turning them into subcommand as well, and add `sign-as` named argument for the signer id, so it is not unnamed there, and rename `network` to `network-config` as there could be more than one network config using different RPCs to a single NEAR network:

```
./near-cli contract \
    call-function \
    as-transaction 'turbo.volodymyr.testnet' 'rate'
    json-args '{"other_user":"volodymyr.testnet", "vote":5}' \
    prepaid-gas '3 Tgas' \
    attached-deposit '1 NEAR' \
    sign-as fro_volod.testnet \
    network-config pagoda-testnet \
    sign-with-keychain \
    send
```

**Comment by agileurbanite** [comment]
👍 shouldn't this be supported in the node version as well? https://github.com/near/near-cli

**Comment by frol** [comment]
Feel free to submit a feature request there. I'm not sure about their path forward

**Comment by frol** [comment]
Resolved in #117

**Comment by frol** [comment]
@agileurbanite [0.2.0-pre.2 release](https://github.com/near/near-cli-rs/releases/tag/0.2.0-pre.2) now allows users to select how to pass function arguments (plaintext JSON, base64-encoded string, or read from file)
